import java.util.*;
public class Practice2 {
    public static void start(){
       try {
           System.out.println("Inside Try block");
           throw new RuntimeException("test");
       }catch (Exception e){
           System.out.println("Inside Catch Block");
           System.out.println(e);
       }
       finally {
           System.out.println("Inside finally block");
       }
    }
    public static void main(String[] args){
        start();
    }
}
